pages/homePage/homePage.css

```diff
@@ -0,0 +1,204 @@
+@import "/styles.css";
+
+.page-wrap {
+
+    grid-template-areas:
+        'p_nav p_nav p_nav p_nav p_nav p_nav'
+        'p_content p_content p_content p_content p_content p_content'
+        'p_content p_content p_content p_content p_content p_content'
+    ;
+}
+
+/* Header */
+.page-header {
+    grid-area: p_header;
+}
+
+.page-sidebar {
+    display: none;
+}
+
+.page-footer {
+    display: none;
+}
+
+.main-content {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    /* align-items: center; */
+    /* justify-content: center; */
+    flex-direction: column;
+
+}
+
+.main-content__header {
+    /* background-color: chartreuse; */
+    height: 100%;
+}
+
+.promo-catagory-holder {
+    display: flex;
+    width: 100%;
+    height: 50%;
+    gap: 2rem;
+    /* background-color: chocolate; */
+    padding: 0.8rem
+}
+
+.promo-catagory {
+    height: 100%;
+    width: auto;
+    aspect-ratio: 1/1;
+    /* background-color: blue; */
+    margin: auto;
+    overflow: hidden;
+}
+
+.promo-catagory>img {
+    width: 100%;
+    height: auto;
+    transition: all .2s ease-in-out;
+}
+
+.promo-catagory>img:hover {
+    transform: scale(1.1);
+    opacity: 50%;
+
+}
+
+.promo-text {}
+
+
+
+/* Carousal */
+
+/* Slideshow container */
+.slideshow-container {
+    height: 100%;
+    position: relative;
+    margin: auto;
+}
+
+/* Hide the images by default */
+.mySlides {
+    display: none;
+    height: 100%;
+}
+
+.mySlides>img {
+    height: 100%;
+    width: 100%;
+    object-fit: none;
+}
+
+/* Next & previous buttons */
+.prev,
+.next {
+    cursor: pointer;
+    position: absolute;
+    top: 50%;
+    width: auto;
+    margin-top: -2.2rem;
+    padding: 1.6rem;
+    color: black;
+    font-weight: bold;
+    font-size: 1.8rem;
+    transition: 0.6s ease;
+    border-radius: 0 3px 3px 0;
+    user-select: none;
+}
+
+/* Position the "next button" to the right */
+.next {
+    right: 0;
+    border-radius: 3px 0 0 3px;
+}
+
+/* On hover, add a black background color with a little bit see-through */
+.prev:hover,
+.next:hover {
+    background-color: rgba(0, 0, 0, 0.8);
+    color: white;
+}
+
+/* Caption text */
+.text {
+    color: #f2f2f2;
+    font-size: 1.5rem;
+    padding: 0.8rem 1.2rem;
+    position: absolute;
+    bottom: 0.8rem;
+    width: 100%;
+    text-align: center;
+}
+
+/* Number text (1/3 etc) */
+.numbertext {
+    color: #f2f2f2;
+    font-size: 1.2rem;
+    padding: 0.8rem 1.2rem;
+    position: absolute;
+    top: 0rem;
+}
+
+/* varundev */
+.carousal_navigator {
+    position: absolute;
+    bottom: 0rem;
+    width: 100%;
+}
+
+/* The dots/bullets/indicators */
+.dot {
+    cursor: pointer;
+    height: 1.5rem;
+    width: 1.5rem;
+    margin: 0 2px;
```

Comment:
I saw you use units for 0 too so add it here also to stay consistent

Reply:
Done.